trezor-crypto/include/TrezorCrypto/bip39.h

```diff
@@ -34,10 +34,10 @@ extern "C" {
 #define BIP39_WORDS 2048
 #define BIP39_PBKDF2_ROUNDS 2048
```

Comment:
should we define statics for max and min for bip39 words?

Reply:
0xa36fD8fF0524f287422337d99bABdf05923f581B